xhr.bs

```diff
@@ -2031,6 +2039,19 @@ attributes initialized to false, so it is suggested that for consistency all
 {{ProgressEvent}} interface do the same.
 
 
+<h3 id=feature-policy>Feature Policy Integration</h3>
+
+<p>This specification defines a policy-controlled feature named <dfn>Synchronous
+XMLHttpRequest</dfn>.
+
+<p>The feature name for <a>Synchronous XMLHttpRequest</a> is "sync-xhr".
+
+<p>The default allowlist for Synchronous XMLHttpRequest is <code>*</code>.
```

Comment:
xref "default allowlist" (elsewhere we use safelist for things like this, is this different enough to use a new name?)

Reply:
I hadn't noticed 'safelist' being used in this context, and I'm not sure if 'safety' is the concept we're trying to convey in any case.

Originally the feature policy spec used 'whitelist', and early reviews suggested we switch to 'allowlist' to match some other specs (I don't recall which; perhaps Web Authentication?)

I'm sure this is open to change, but that should probably happen on the FP issues list :)

Reply:
WebAuthn doesn't seem to use it.

Reply:
https://github.com/w3c/webauthn/issues/327 -- apparently not anymore (I still don't remember if this was the spec that was brought up, could be a complete coincidence)